packages/plugin-table/test/Table.test.ts

```diff
@@ -3667,22 +3667,203 @@ describePlugin(Table, async testEditor => {
     describe('mergeCells', async () => {
         it('should merge two adjacent cells together', async () => {
             await testEditor(BasicEditor, {
-                contentBefore: '<table><tbody><tr><td>a[b</td><td>c]d</td></tr></tbody></table>',
+                /* eslint-disable prettier/prettier */
```

Comment:
boooooh

Reply:
What? You prefer a gross and intrusive regular expression that removes the space? Or the whole html inline perhaps? :trollface:

Reply:
Sincerely ? Yes I prefer the regex because at least the html in the test is easy to update. In your version, I have to manually manage the quotes and commas, while they do not even exist with the regex. With the regex, it's just like if you were in a proper html file, nothing else to worry about. You wrote a lot of tests though and it would be a waste of precious resources to have you update them (and to be fair, this regex is not even merged yet !) so we'll leave it at that for this PR.

Reply:
Ok. Agree to disagree ^^